rif/relay/index.md

```diff
@@ -5,10 +5,11 @@ tags: rif, relay, gas station network, gsn
 permalink: /rif/relay/
 ---
 
-RIF Relay is a secure sponsored transaction system that enables users to pay fees using ERC-20 tokens.
+RIF Relay is a secure sponsored transaction system that enables users to pay the transaction fees using ERC-20 tokens.
```

Comment:
```suggestion
RIF Relay is a secure sponsored transaction system that enables users to pay the transaction fees using ERC-20 tokens.
This enables end users to transact entirely using one asset instead of having to manage a separate asset for gas.
```

Reply:
updated